packages/lib/server/repository/user.ts

```diff
@@ -664,25 +664,18 @@ export class UserRepository {
     return !!teams.length;
   }
   static async isAdminOrOwnerOfTeam({ userId, teamId }: { userId: number; teamId: number }) {
-    const team = await prisma.team.findUnique({
+    const isAdminOrOwnerOfTeam = await prisma.membership.findFirst({
```

Comment:
We query the membership table directly. The team relationship is not adding here.